### PR TITLE
Remove non-inclusive language

### PR DESCRIPTION
**Pull request by neonichu**
rdar://problem/69396992

### CODE REVIEW

**Comment by neonichu** [comment]
@swift-ci please test

**Comment by neonichu** [comment]
We need a corresponding PR on SwiftPM before merging.

File: Sources/TSCUtility/PkgConfig.swift [hunk 1]
```diff
@@ -14,16 +14,16 @@ import Foundation
...
     public var description: String {
         switch self {
         case .couldNotFindConfigFile(let name):
             return "couldn't find pc file for \(name)"
         case .parsingError(let error):
             return "parsing error(s): \(error)"
-        case .nonWhitelistedFlags(let flags):
-            return "non whitelisted flag(s): \(flags)"
+        case .nonAllowlistedFlags(let flags):
+            return "not allowed flag(s): \(flags)"
```
Comment (abertelrud) [inline Sources/TSCUtility/PkgConfig.swift:26]: Maybe a nitpick, but would "prohibited flags" make more sense here?

**Review by abertelrud — APPROVED** [review]

File: Sources/TSCUtility/PkgConfig.swift (hunk 1 above)
Comment (neonichu) [reply Sources/TSCUtility/PkgConfig.swift:26]: No, that's good. I was thinking "nonAllowlisted" sounds a bit strange, but didn't come up with a better alternative.

**Comment by neonichu** [comment]
@swift-ci please test

**Comment by neonichu** [comment]
@swift-ci please test

**Comment by neonichu** [comment]
@swift-ci please test

**Comment by neonichu** [comment]
@swift-ci please test

**Comment by neonichu** [comment]
@swift-ci please test

**Review by abertelrud — APPROVED** [review]

**Comment by compnerd** [comment]
@neonichu this is breaking the Windows builds of swift-package-manager since apple/swift-package-manager#2954 isn't merged :-(

**Comment by neonichu** [comment]
Sorry, I had to wait for the final smoke test run to complete. Unfortunately we don't have a good way to merge two PRs together because of how the self-hosted job works. Hopefully we can resolve this type of issue as part of getting rid of the vendored TSC.

**Comment by neonichu** [comment]
Actually, it seems like the real issue is that the self-hosted jobs have been set to "Required". The original intent was apparently to have them for faster CI iteration, but only smoke tests should be required for merging.

**Comment by neonichu** [comment]
OK, this is fixed now. Only the "smoke tests" jobs are required, so in the future we can merge TSC and SwiftPM PRs together.

**Pull request merged**